stable/influxdb/Chart.yaml

```diff
@@ -1,20 +1,14 @@
 apiVersion: v1
 name: influxdb
-version: 4.3.1
+version: 4.3.2
 appVersion: 1.7.9
-description: Scalable datastore for metrics, events, and real-time analytics.
+deprecated: true
+description: DEPRECATED Scalable datastore for metrics, events, and real-time analytics.
```

Comment:
is there a new home?

Reply:
Yep: https://github.com/helm/charts/blob/master/stable/influxdb/README.md#deprecated---influxdb

Already in the README from previous commit.